ew/juviecmd.py

```diff
@@ -392,6 +392,7 @@ async def mine(cmd):
 					str_event_start = event_def.str_event_start
 					if event_data.event_type == ewcfg.event_type_minecollapse:
 						str_event_start = str_event_start.format(cmd = ewcfg.cmd_mine, captcha = ewutils.text_to_regional_indicator(event_data.event_props.get('captcha')))
+						event_data.time_expir = time_now + 65
```

Comment:
I don't think this will work, for two reasons:
1. the change needs to happen after the message has been sent, in order to solve #71 
2. the change needs to be persisted in the database to take effect

Reply:
Sorry, I patched this late at night - I've now got it now addressing both with a response it sends immediately instead of waiting for the end of the mine command. For the sake of an extra response in the channel, we'll give players maximum time to address it.